browser-test/browser-test-compose.yml

```diff
@@ -25,9 +25,9 @@ services:
       - IDCS_DISCOVERY_URI=http://dev-oidc:3390/.well-known/openid-configuration
       - APPLICANT_REGISTER_URI=http://dev-oidc:3390/
       - DB_JDBC_STRING=jdbc:postgresql://database:5432/postgres
-      - BASE_URL=http://civiform:9000
+      - BASE_URL=${BASE_URL:-http://civiform:9000}
```

Comment:
For my own understanding, why does `BASE_URL` get declared twice here?

Reply:
This lets us override the `BASE_URL` env var in the container.  If it's not set, then we default it to `http://civiform:9000`.  This lets us set BASE_URL to something else (`http://localhost:9999`) for the local browser tests.

Reply:
I think to split it out into a separate file, we will either need to pass a flag into `run-browser-test-env` anyway to tell it to run in local mode or add something to `truth.sh` to detect when a variable is set and not exit, as we'll be wanting to set a variable in `run-browser-test-env-local`.  What do you think?